server/views/pages/analytics/protectedCharacteristicTemplate.njk

```diff
@@ -23,6 +26,27 @@
     </div>
   </div>
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form id="form-select-characteristic" method="get" novalidate>
+        <div class="govuk-form-group">
+          <label class="govuk-label" for="sort">
+            Select a protected characteristic
+          </label>
+          <select class="govuk-select" id="characteristic" name="characteristic">
+              {%- for option in characteristicOptions -%}
+                <option value="{{ option.value }}" {%- if option.selected -%}selected{%- endif -%}>{{ option.label }}</option>
```

Comment:
you _don't_ want to trim whitespace around the `selected` attribute otherwise it'll run together with the `value` attribute:

`<option value="age"selected>Age</option>`